### PR TITLE
Split out the list of files to put in documentation

### PR DESCRIPTION
**Pull request by dkfellows**
See https://github.com/SpiNNakerManchester/SpiNNUtils/pull/143

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -370,10 +370,34 @@
...
+# Do the rst generation; remove files which aren't in git first!
+for fl in os.listdir("."):
+    if (os.path.isfile(fl) and fl.endswith(".rst") and
+            fl not in ("index.rst", "modules.rst")):
+        os.remove(fl)
+os.chdir("../..")  # WARNING! RELATIVE FILENAMES CHANGE MEANING HERE!
+apidoc.main([
+    '-o', _output_dir, _package_base,
+    # No examples at all
+    "spinnaker_graph_front_end/examples",
```
Comment (Christian-B) [inline doc/source/conf.py:402]: Example need to be move in a later PR

**Review by Christian-B — APPROVED** [review]
ok to merge

File: doc/source/conf.py (hunk 1 above)
Comment (dkfellows) [reply doc/source/conf.py:402]: Was in #178

**Review by Christian-B — APPROVED** [review]
ok to merge

**Pull request merged**